keyboards/handwired/maple60/readme.md

```diff
@@ -0,0 +1,27 @@
+# maple60
+
+A handwired 60% keyboard with maple case.
+
+<img src="https://user-images.githubusercontent.com/6307033/151084291-9d7a800d-4619-462d-a24c-768d7e9213cc.jpg" width="200"> <img src="https://user-images.githubusercontent.com/6307033/151084297-91f25027-c272-4aa3-ad2b-c35896eb2a2d.jpg" width="200">
```

Comment:
Please remove the html tags, and use markdown for the images (eg `![](image_url.jpg)`, and host the image on imgur.